README.md

```diff
@@ -1,4 +1,37 @@
-# opg-metrics
+# OPG Metrics
+
+## Integration and Mocking
+
+Within the solution we have provided a [swagger](/terraform/environment/api/swagger.json) doc and [example packet](/terraform/environment/api/metrics.json) for help in understanding the integration.
```

Comment:
Wonder if this OpenAPI spec is in the right location, /docs/ folder maybe or app level. Not sure TF env is the place.

Reply:
I would usually agree however the same spec is used to create the API Gateway endpoint and want to not duplicate the code and also leave it in the appropriate area.

Also I keep refereeing to it as Swagger, do you think OpenAPI would be more appropriate in the description as it is the spec rather than the brand name?

Reply:
I'd go with OpenAPI as it's the standard, Swagger is the tools. https://swagger.io/blog/api-strategy/difference-between-swagger-and-openapi/ for a summary.

Reply:
Ah, code location makes sense then. May need a way to wayfind it from the repo root/app, but I guess that's what our new repo docs standards are for!